src/Compiler/OnnxMlirCompiler.cpp

```diff
@@ -49,6 +53,50 @@ ONNX_MLIR_EXPORT const char *omGetCompilerOption(const OptionKind kind) {
   return strdup(val.c_str());
 }
 
+#ifdef _WIN32
```

Comment:
@sstamenova I know its already pushed, if you have a preferred way to handle MS custom handling, please let me know and I will integrate in a follow up cleanup PR.

Reply:
My recommendation would be for any situation like this to first look for the equivalent function in llvm - they almost always exist and they are properly implemented cross-platform, so we wouldn't need any windows specific handling.

If that can't be avoided, something like what you have here generally works, but in this particular case, I think there is an llvm function.